### ISSUE TITLE
"Import Module" should add a @Folder annotation

### ISSUE THREAD

**Issue by retailcoder** [rubberduck-vba/Rubberduck#4282]
ref. #3332 

When right-clicking a folder in the CE and selecting "import", the imported module gets imported under the default folder unless it already has a `@Folder` annotation, in which case it gets added to whatever folder that annotation is specifying... which may or may not make sense in the current project.

Let's make the "import" functionality edit any existing `@Folder` annotation to match the selected CE folder, or add one such annotation when there's none.

**Comment by bclothier** [comment]
Closing this in favor of #1777 - track the issue there.

**Comment by mansellan** [comment]
Reopening, #1777 was for Project Explorer not Code Explorer.

**Comment by daFreeMan** [comment]
I'm not the best at GitHub, but didn't @IvenBach close this back in Oct '18 with a PR that specifically fixed it?

**Comment by IvenBach** [comment]
No. That PR was closed [due to age/staleness.](https://github.com/rubberduck-vba/Rubberduck/pull/4444#issuecomment-451313833). I never got back around to completing it.
